sky/backends/backend_utils.py

```diff
@@ -327,7 +327,7 @@ def wrap_file_mount(cls, path: str) -> str:
     def make_safe_symlink_command(cls, *, source: str, target: str) -> str:
         """Returns a command that safely symlinks 'source' to 'target'.
 
-        All intermediate directories of 'source' will be owned by $USER,
+        All intermediate directories of 'source' will be owned by $(whoami),
```

Comment:
Is there a specific reason to change from `$USER` to `$(whoami)`?

Reply:
Some of images do not have $USER set, but most of the time `$whoami` will work